.github/workflows/integration-tests.yml

```diff
@@ -609,3 +609,27 @@ jobs:
           ${SCCACHE_PATH} --show-stats
 
           ${SCCACHE_PATH} --show-stats | grep -e "Cache hits\s*[1-9]"
+
+  rust-test-coverage:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: integration-tests
+          path: /home/runner/.cargo/bin/
+      - name: Chmod for binary
+        run: chmod +x ${SCCACHE_PATH}
+
+      - name: Coverage test
+        run: |
+          rustup toolchain install nightly
+          cargo new hello
+          cd hello
+          echo "serde = { version = \"1.0\", features = [\"derive\"] }" >> Cargo.toml
+          cargo +nightly test
```

Comment:
can you please add a check that we are indeed caching the right things?

see the other integration tests

Reply:
sorry for the incorrect path. fixed it in [10b37a7](https://github.com/mozilla/sccache/pull/1740/commits/10b37a7c4ea304d40577c21c35814b936146f5e5)

and check it in my fork repo. it succeed
<img width="1152" alt="Screenshot 2023-04-21 at 6 30 08 PM" src="https://user-images.githubusercontent.com/8209234/233614309-5213d194-3505-4671-a50b-b25346e733ff.png">